src/v2i-hub/MUSTSensorDriverPlugin/src/MUSTSensorDriverPlugin.cpp

```diff
@@ -22,6 +22,10 @@ namespace MUSTSensorDriverPlugin {
 	
 	MUSTSensorDriverPlugin::MUSTSensorDriverPlugin(const string &name): PluginClientClockAware(name)
 	{
+		if (isSimulationMode()) {
+			// Subscribe to the TimeSyncMessage
+			SubscribeToMessages();
```

Comment:
If it is not in sim mode, does this must driver subscribe to message at all?

Reply:
Is there a way to check if there are filters exist, then subscribe whenever it detect at least one filter exist?

Reply:
Not that I am aware of.  This condition only needs to exist for plugins that are integrated with simulation but do not by default subscribe to any message filters.